tools/build_script_generator/common.py

```diff
@@ -206,6 +206,7 @@ def common_compiler_flags(compiler, target):
         "-Wlogical-op",
         "-Wpointer-arith",
         "-Wundef",
+        "-Wno-redundant-decls",
```

Comment:
This is an issue for a lot of external code, and I don't see this as a useful warning.